### PR TITLE
Small adjustments to the README

### PR DESCRIPTION
**Pull request by waldyrious**
- Link to ghq, break its line and add missing punctuation; 
- Tweak wording in Requirements section for consistency;
- Clarify that the username is passed as the second parameter.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +27,7 @@ gh fuzzyclone
 ```
 
 By default, the extension only fetches repositories under your username.
-To get other repositories, you can pass in a username, like so:
+To get other repositories, you can pass in a username as a second parameter, like so:
```
Comment (hashue) [inline README.md:30]: It is not always username as second parameter.

Comment (hashue) [inline README.md:30]: It is not always username as second parameter.

File: README.md [hunk 2]
```diff
@@ -3,13 +3,15 @@
 A [gh](https://github.com/cli/cli) extension to clone GitHub repositories using
 [fzf](https://github.com/junegunn/fzf#readme).
 
-If you wanna use `ghq`, I recommend a very nice extension [gh-q](https://github.com/kawarimidoll/gh-q/) made by [kawarimidoll](https://github.com/kawarimidoll)
+If you want to use [`ghq`](https://github.com/x-motemen/ghq),
```
Comment (hashue) [inline README.md:6]: I think unnecessary ghq link 'cuase not use ghq in my plugins

**Review by hashue — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (waldyrious) [reply README.md:30]: Oh, interesting. Can you help clarify with an example or two where that's not the case?

File: README.md (hunk 2 above)
Comment (waldyrious) [reply README.md:6]: It's just so people will know what you mean when you refer to ghq. The link itself only provides information if someone happens to not be familiar with ghq when they come across this repo and read the README (that was my case, for example). Those who already know what ghq is can simply ignore the link, so as I see it, there's no harm in adding it.

File: README.md (hunk 1 above)
Comment (hashue) [reply README.md:30]: @waldyrious Sorry for late response.

Please check this PR.
https://github.com/hashue/gh-fuzzyclone/pull/1

Comment (waldyrious) [reply README.md:30]: I think I see what you mean. How about this, then?

> To get other repositories, you can pass in a username as an additional parameter, like so:
> ```
> gh fuzzyclone <username> <repository>
> ```

Comment (hashue) [reply README.md:30]: Sounds great！！！
I'll merge this change applied！

Comment (waldyrious) [reply README.md]: Cool! I've applied the change :)

**Comment by hashue** [comment]
@waldyrious Cool！I merged.

**Pull request merged**